composer.json

```diff
@@ -18,24 +18,24 @@
         "issues": "https://github.com/stwe/DatatablesBundle/issues"
     },
     "require": {
-        "php": ">=7.1",
+        "php": ">=7.2|>=8.0",
         "doctrine/orm": "^2.5",
-        "friendsofsymfony/jsrouting-bundle": "^1.6|^2.0|^3.0",
-        "symfony/config": "^3.4|^4.1|^5.0",
-        "symfony/dependency-injection": "^3.4|^4.1|^5.0",
-        "symfony/http-foundation": "^3.4|^4.1|^5.0",
-        "symfony/http-kernel": "^3.4|^4.1|^5.0",
-        "symfony/framework-bundle": "^3.4|^4.1|^5.0",
-        "symfony/options-resolver": "^3.4|^4.1|^5.0",
-        "symfony/property-access": "^3.4|^4.1|^5.0",
-        "symfony/routing": "^3.4|^4.1|^5.0",
-        "symfony/security-core": "^3.4|^4.1|^5.0",
-        "symfony/translation": "^3.4|^4.1|^5.0",
+        "friendsofsymfony/jsrouting-bundle": "^2.0|^3.0",
+        "symfony/config": "^4.4|^5.4|^6.0",
+        "symfony/dependency-injection": "^4.4|^5.4|^6.0",
+        "symfony/http-foundation": "^4.4|^5.4|^6.0",
+        "symfony/http-kernel": "^4.4|^5.4|^6.0",
+        "symfony/framework-bundle": "^4.4|^5.4|^6.0",
+        "symfony/options-resolver": "^4.4|^5.4|^6.0",
+        "symfony/property-access": "^4.4|^5.4|^6.0",
+        "symfony/routing": "^4.4|^5.4|^6.0",
+        "symfony/security-core": "^4.4|^5.4|^6.0",
+        "symfony/translation": "^4.4|^5.4|^6.0",
         "twig/twig": "^2.9|^3.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^7.5|^8.5",
-        "friendsofphp/php-cs-fixer": "^2.15"
+        "phpunit/phpunit": "^8.5|^9.5",
+        "friendsofphp/php-cs-fixer": "^3.0"
```

Comment:
If you upgrade PHP-cs-fixer, you need to upgrade the config file `.php_cs.dist` to the new format `.php-cs-fixer.dist.php`

Reply:
Do you also prefer that I commit al changes made by php-cs-fixer when running?
It changes 13 files unrelated to my modifications but due to updated rules.

Reply:
IMO it should be better to upgrade php-cs-fixer in a separate PR first with all related changes
And then this PR only for PHP 8 / SF 6 upgrade

Reply:
Agreed that if possible that should be better.. however php-cs-fixer ^2 doesn't support Symfony 6 at all, so composer won't install Symfony 6 if I keep composer ^2 and I couldn't create this PR :-)

Reply:
Yes but I will merge the other PR before this one

Reply:
@Seb33300
https://github.com/stwe/DatatablesBundle/pull/984